backend/community/guides/permissions.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import permissions
+
+
+class IsOwnerOrPublic(permissions.BasePermission):
+
+    def has_object_permission(self, request, view, obj):
+        return obj.owner == request.user or not obj.private
```

Comment:
Here I can see the `owner` is assumed on the object. This is probably where you meant to put the comment from the other permissions file?

Reply:
The previous comment was a left-over from copying the code from DRF internals.
I think the code is pretty self explanatory and requires no comment at all.